Combobox.js

```diff
@@ -575,22 +571,50 @@ define([
 			return new ComboPopup({combobox: this});
 		},
 
+		evaluateFilteringConditions: function (inputElement) {
+			var mobile = this._useCenteredDropDown();
+			if (this.minFilterChars !== inputElement.value.length !== 0 &&
```

Comment:
The code seems to work but I'm confused about this condition for a number of reasons.  I wonder if you could make it clearer.

1. `this.minFilterChars !== inputElement.value.length`: It seems like you should be testing for `minFilterChars > inputElement.value.length` or something like that.   Why the `!==`?  In other words, what's special about the case where `minFilterChars` and `inputElement.value.length` are exactly the same?
2. I don't how a condition with two `!==` works.  For example, if `minFilterChars` is 0 and `inputElement.value.length` is 1, the condition becomes `0 !== 1 !== 0`, but is that true or false?  It's quite unusual to write a condition like this so maybe you can rewrite using an `&&`?